blueprint/core/blueprint_ScrollView.h

```diff
@@ -23,25 +23,70 @@ namespace blueprint
     class ScrollView : public View
     {
     public:
+        //==============================================================================
+        // Props following CSS Scrollbars spec with some omissions/additions.
+        // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars
+        static inline juce::Identifier overflowProp       = "overflow";
```

Comment:
So I think our general stance on native props should be to make a minimal set of very concrete props, and defer all composition and clever syntax up to the javascript layer.

For example, native never needs to know about the idea of this `overflow` prop. At the `ScrollView` layer in the js library, we can catch an `overflow` prop and transform it to the necessary composition of `overflow-x` and `overflow-y`.

Reply:
Yep so lose `overflowProp` here and maintain the minimal `overflowXProp` and `overflowYProp`. JS converts `overflow` to desired composition for C++/native?

Reply:
Yup exactly